sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet/gluoncv_ssd_mobilenet_neo.ipynb

```diff
@@ -389,12 +238,25 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "compiled_model = ssd_estimator.compile_model(target_instance_family='ml_p3', \n",
-    "                                             input_shape={'data':[1, 3, 512, 512]},\n",
-    "                                             output_path=s3_output_location,\n",
-    "                                             framework='mxnet', \n",
-    "                                             framework_version='1.4.1'\n",
-    "                                            )"
+    "%%time\n",
```

Comment:
Not sure of a URL changed, but downloading data seems to be failing.  Here's the GitHub Issue: https://github.com/aws/amazon-sagemaker-examples/issues/1615

Reply:
It does work sometimes/days. I had added an alternate link in the notebook in a markdown cell. Have made it much more readable now.